src/utils/search-query-transform.js

```diff
@@ -216,3 +216,12 @@ export const queryStringToQueryData = (queryString) => {
 
   return queryDataObject
 }
+
+export const areQueriesEqual = (oldQuery, newQuery) => {
+  for (let key of Object.keys(oldQuery)) {
+    if (oldQuery[key] !== newQuery[key]) {
+      return false
+    }
+  }
+  return true
+}
```

Comment:
Maybe this doesn't matter for queries but this wouldn't work if `newQuery` has keys that aren't present in `oldQuery`. For example, this will return `true` for:

```js
areQueriesEqual({}, { foo: 'bar' })
```

We could use `lodash.isEqual` or else do something like is described in [this SO answer](https://stackoverflow.com/a/35951373) after converting both arguments to Map objects or adapting that solution to just work on regular objects and avoid the conversion.

Reply:
@sarayourfriend, I added `lodash.isEqual`. I was using this comparison because the queries were supposed to have the same structure, and the same number of properties. In fact, we probably won't need this comparison after the All content page is actually different from the Image page.

Reply:
Gotcha! I figured that maybe this specific consideration didn't matter for queries for that reason. Either way works for me!